documentation/modules/mtv-workflow.adoc

```diff
@@ -20,11 +20,14 @@ The workflow describes the following steps:
 
 . You create a source provider, a target provider, a network mapping, and a storage mapping.
 . You create a migration plan that includes the following resources:
+
 * Source provider
 * Target provider
 * Network mapping
 * Storage mapping
 * One or more virtual machines (VMs)
 
 . You run a migration plan by creating a `Migration` CR that references the `Plan` CR. If a migration is incomplete, you can run a migration plan multiple times until all VMs are migrated.
-. For each VM in the migration plan, the Migration Controller creates a `VirtualMachineImport` CR and monitors its status. When all VMs have been migrated, the Migration Controller sets the status of the migration plan to `Completed`. The power state of a source VM is maintained after migration.
+. For each VM in the migration plan, the `Migration Controller` service creates a `VirtualMachine` CR and records the progress of the migration in its `Status` subresource.
++
```

Comment:
What is this `Status` subresource?
Is it in the `Migration Controller` service?
Because if it is the target VM CR - I cannot see under its status field something related to the migration progress

Reply:
@mansam Where is the progress of the VM migration status recorded? Is it in the Migration CR or somewhere else?

Reply:
@mguetta1 I updated this sentence based on Sam's comment on Slack. Please check. THanks

Reply:
Looks good to me